syntax = "proto3";
package messageService;

option csharp_namespace = "Il2CppToolkit.Interop";

message MessageRequest {
	string msg = 1;
}

message MessageReply {
	string reply = 1;
}

service MessageService {
	rpc SendMessage (MessageRequest) returns (MessageReply){}
}